src/v2/Apps/_WorksForYou2/Components/WorksForYou2ArtistFeed.tsx

```diff
@@ -0,0 +1,156 @@
+import { Box, Button, EntityHeader, Spacer } from "@artsy/palette"
+import { useState } from "react"
+import {
+  createPaginationContainer,
+  graphql,
+  RelayPaginationProp,
+} from "react-relay"
+import ArtworkGrid from "v2/Components/ArtworkGrid"
+import { useSystemContext } from "v2/System"
+import { WorksForYou2ArtistFeed_viewer } from "v2/__generated__/WorksForYou2ArtistFeed_viewer.graphql"
+
+interface WorksForYou2ArtistFeedProps {
+  relay: RelayPaginationProp
+  viewer: WorksForYou2ArtistFeed_viewer
+}
+
+const WorksForYou2ArtistFeed: React.FC<WorksForYou2ArtistFeedProps> = ({
+  relay,
+  viewer,
+}) => {
+  const { user } = useSystemContext()
+  const [isLoading, setLoading] = useState(false)
+  const { artist } = viewer
+
+  if (!artist) {
+    return null
+  }
+
+  const avatarImageUrl = artist?.image?.resized?.url
+  const meta = artist?.counts?.forSaleArtworks?.toLocaleString()
+  const worksForSaleHref = artist.href + "/works-for-sale"
+
+  const handleLoadMore = () => {
+    if (!relay.hasMore() || relay.isLoading()) return
+
+    setLoading(true)
+
+    relay.loadMore(25, err => {
+      if (err) console.error(err)
+      setLoading(false)
+    })
+  }
+
+  return (
+    <>
+      <EntityHeader
+        name={artist.name!}
+        meta={meta}
+        imageUrl={avatarImageUrl}
```

Comment:
Instead of giving this `imageUrl` pass it `image` with an object containing a `src`, `srcSet`, and `lazyLoad`